staging/src/k8s.io/component-helpers/apimachinery/lease/controller.go

```diff
@@ -221,13 +221,14 @@ func (c *controller) newLease(base *coordinationv1.Lease) (*coordinationv1.Lease
 				Namespace: c.leaseNamespace,
 			},
 			Spec: coordinationv1.LeaseSpec{
-				HolderIdentity:       pointer.StringPtr(c.holderIdentity),
-				LeaseDurationSeconds: pointer.Int32Ptr(c.leaseDurationSeconds),
+				HolderIdentity: ptr.To(c.holderIdentity),
```

Comment:
I believe we should also move this outside of the if/else, but I'll let a reviewer weigh in.

Reply:
I'm indifferent since we should be guaranteed to be the holder on the update case.

Reply:
ack, thanks!